packages/zowe-explorer/README.md

```diff
@@ -23,26 +23,24 @@ More information:
 
 ## Contents
 
-- [What's new in Zowe Explorer 1.11.0](#whats-new-in-zowe-explorer-1.11.0)
+- [What's new in Zowe Explorer 1.12.1](#whats-new-in-zowe-explorer-1.12.1)
 - [Prerequisites](#prerequisites)
 - [Getting Started](#getting-started)
 - [Sample Use Cases](#sample-use-cases)
 - [Credentials Security](#credentials-security)
 - [Usage Tips](#usage-tips)
 - [Extending Zowe Explorer](#extending-zowe-explorer)
 
-## What's new in Zowe Explorer 1.11.0
-
-New features:
-
-- Added login and logout functions for base profiles. You can now log in to API Mediation Layer and generate a token for your base profile.
+## What's new in Zowe Explorer 1.12.1
 
 Bug Fixes:
 
-- Fixed the empty profile folders in Favorites issue.
-- Fixed the initialization error that occurred when base profiles were used while being logged out from API ML.
-- Fixed the issue preventing the tree refresh function from updating extender profiles.
-- Fixed the issue causing jobs retrieval failure when using profiles with tokens.
+- Fixed the issue that prevented edited profile base paths from being saved.
+- Fixed the issue that prevented Zowe Explorer from storing empty values for optional profile fields, such as `user`, `password`, `timeout`, and `encoding`. This is done to be consistent with the way Zowe CLI stores profile information when creating and editing profiles.
+- Fixed the issue that caused repeated credential prompting if a user refused to authenticate.
+- Fixed the issue that caused removed favorite profiles to persist between IDE sessions.
+- Fixed the issue that prevented the editing of data set attributes on allocation functionality from being enabled.
```

Comment:
I don't think this belongs here

Reply:
Thank you for catching this one, Richelle! Let me remove it.